bigbluebutton-html5/imports/ui/components/loading-screen/component.jsx

```diff
@@ -6,7 +6,6 @@ const LoadingScreen = ({ children }) => (
     <div className={styles.spinner}>
       <div className={styles.bounce1} />
       <div className={styles.bounce2} />
-      <div className={styles.bounce3} />
```

Comment:
Why remove that, is for loading has only two bounces?

Reply:
There was a css error for this style class not existing. Wasn't sure if it was still needed.

Reply:
it's necessary, stay a strange animation with only two. that class can removed by mistake.